CreatubblesAPIClient/Sources/Model/Creation.swift

```diff
@@ -66,6 +66,7 @@ open class Creation: NSObject, Identifiable {
 
     open let objFileUrl: String?
     open let playIFrameUrl: String?
+    open let playIFrameUrlIsMobileReady: Bool?
```

Comment:
Personally, I'd avoid optional boolean properties and use default value (probably `false` here).